Libs/GObject/Structs/Type.cs

```diff
@@ -49,6 +49,25 @@ public Type(ulong value)
 
         #endregion
 
+        // Print out the name of the gtype (for debugging purposes)
+        public override string ToString()
+        {
+            return Marshal.PtrToStringAnsi(Global.Native.type_name(_value));
+        }
+
+        public override bool Equals(object obj)
```

Comment:
Do we have to override Equals? The type class is already a struct which is comparing its values automatically this should be obsolete?

Reply:
Huh, didn't think you could do that.

That's much simpler, I've gone ahead and removed `Equals` 👍